patches/@react-native-camera-roll+camera-roll+7.4.0.patch

```diff
@@ -0,0 +1,54 @@
+diff --git a/node_modules/@react-native-camera-roll/camera-roll/android/build.gradle b/node_modules/@react-native-camera-roll/camera-roll/android/build.gradle
+index 6891fa3..3759bed 100644
+--- a/node_modules/@react-native-camera-roll/camera-roll/android/build.gradle
++++ b/node_modules/@react-native-camera-roll/camera-roll/android/build.gradle
+@@ -81,7 +81,9 @@ def findNodeModulePath(baseDir, packageName) {
+ }
+ 
+ def resolveReactNativeDirectory() {
+-    def reactNative = file("${findNodeModulePath(rootProject.projectDir, "react-native")}")
++    def projectDir = this.hasProperty('reactNativeProject') ? this.reactNativeProject : rootProject.projectDir
++    def modulePath = file(projectDir);
++    def reactNative = file("${findNodeModulePath(modulePath, 'react-native')}")
+     if (reactNative.exists()) {
+         return reactNative
+     }
+diff --git a/node_modules/@react-native-camera-roll/camera-roll/android/src/main/AndroidManifest.xml b/node_modules/@react-native-camera-roll/camera-roll/android/src/main/AndroidManifest.xml
+index dbda53f..98fb413 100644
+--- a/node_modules/@react-native-camera-roll/camera-roll/android/src/main/AndroidManifest.xml
++++ b/node_modules/@react-native-camera-roll/camera-roll/android/src/main/AndroidManifest.xml
+@@ -1,6 +1,6 @@
+ 
+ <manifest xmlns:android="http://schemas.android.com/apk/res/android"
+-          package="com.reactnativecommunity.cameraroll">
++          >
```

Comment:
We don't have this before, what's the reason of this patch?

Reply:
This patch file suggested by @situchan [here](https://github.com/Expensify/App/pull/37221#pullrequestreview-1901716570)

Reply:
@shahinyan11 this change was originally from your large patch. can be removed

Reply:
@situchan I got it